### DISCUSSION TITLE
IRC clients that can run on helloSystem

### DISCUSSION THREAD

**Discussion by mchasard** [helloSystem/hello#498] [General]
Would like to know if under hellosystem i could install an Irc program  ? 

thanks in advance

**Comment by mchasard** [comment]
Its seems that nobody takes times to answer at  this simplest request.

sorry but i'm a newbie here...

**Reply by probonopd** [reply]
Hello @mchasard. Assuming that you are looking for IRC clients that can run on helloSystem, you can find some here:
https://www.freshports.org/irc/

They can all be installed via the `pkg` command. Remember, helloSystem is FreeBSD under the hood.

**Reply by mchasard** [reply]
Thanks for your answer very cool ,  i just install hexchat ... but i have to launch it via search action , is there a way to add it in the menu ?

**Reply by probonopd** [reply]
It _should_ be added automatically...
